docs/WindowsNative.md

```diff
@@ -0,0 +1,64 @@
+# Running on Windows
+
+Contact: [Nicole Mazzuca](https://github.com/ubsan) (if you're at microsoft,
```

Comment:
This extra part about at MS doesnt make much sense for an open source project :)

Reply:
Also perhaps link this from the main readme?

Reply:
Perhaps, instead of being linked, some of this could be written there to give it more visibility (and add a "Further docs at ..." or something?)

Reply:
Ah, yeah, this is from when I was sending it to internal people :P